File: Coiner.Business/Heplers/Product.cs
using Coiner.Business.Models;
using System;
using System.Collections.Generic;
using System.Text;

namespace Coiner.Business.Heplers
{
    public class Product
    {
        public string ProductName { get; set; }

        public decimal LastTransaction { get; set; }
     
        public decimal TotalCapitalization { get; set; }

        public List<decimal> Transactions { get; set; }

        public Project Project { get; set; }

        public decimal TransactionVariation { get; set; }

        public decimal CoinValue { get; set; }

        public decimal TotalCoinNumber { get; set; }

        public string MinBuyValue { get; set; }

        public string MaxSellValue { get; set; }

        public Product()
        {
            Transactions = new List<decimal>();
        }
      
    }
}

File: Coiner/ClientApp/src/app/models/product.ts
import { Project } from "./project";

export class Product {
    productName: string;
    lastTransaction: number;
    totalCapitalization: number;
    transactions: number[];
    project: Project;
    transactionVariation: number;
    coinValue:number;
    totalCoinNumber: number;
    minBuyValue:string;
    maxSellValue:string;
}